plugin-template

```diff
@@ -307,5 +319,27 @@ def check_for_deprecated_files(plugin_root_dir):
         print("No deprecated files found.")
 
 
+def migrate_github_issues(config, plugin_root_dir, verbose=False):
+    """Moves the plugin's issue checker CI/CD off of Redmine and onto GitHub"""
+    config['issue_tracker'] = 'github'
+    name = config['plugin_name']
+    redmine = config['redmine_project']
+    if not redmine:
+        print("Redmine project name not set. Please set 'redmine_project' in template_config.yml"
+              " and rerun this command")
+        return 2
+    config['redmine_project'] = None
+    config['update_redmine'] = False
+    write_template_section(config, "generate_config", plugin_root_dir, verbose=verbose)
+    write_template_section(config, "github", plugin_root_dir, verbose=verbose)
+    print("\nUpdating pyproject.toml to point to GitHub issues")
```

Comment:
Why should a print statement start with a newline?

Reply:
It was done earlier in the script. I changed it to after.